template_tinytemplate/README.md

```diff
@@ -0,0 +1,18 @@
+# template_tinytemplate
+
+See the documentation for the minimalist template engine [tiny_template](https://docs.rs/tinytemplate/1.1.0/tinytemplate/).
+
+## Usage
+
+### Server
+
+```bash
+cd examples/template_tinytemplate
+cargo run (or ``cargo watch -x run``)
```

Comment:
Markdown formatting doesn't work inside of code blocks, and also I think shell code blocks should consist of valid shell commands.

```suggestion
cargo run # (or 'cargo watch -x run')
```